### ISSUE TITLE
[CRITICAL] Update Open-Source communities page

### ISSUE THREAD

**Issue by vipulgupta2048** [asetalias/asetalias.github.io#96]
**Need to update that section with these and any more we can find.**
- Mozilla Delhi Open Community [https://wiki.mozilla.org/India/Delhi] 
- PyData Delhi Community
- Women Who Go Delhi

**Extensive issues on landing pages of those communities**
- ILUG-D has changed its logo (Need to be updated)
- ILUG-D, strangely enough, has the same description as PyDelhi. Change it (Time to git blame)
- PyDelhi [Description updated [here](https://t.me/PyDelhigroup/13211)]
- Improve the Landing page to be in sync with the theme of the entire website [Critical]
- Mozilla Delhi has links same as ILUGD. 

**Basically, the whole page needs a fix/revamp/update**

**Comment by utkarsh2102** [comment]
If correctly inferred, we have to change the data and the logo of some of the organisations, right?

**Comment by vipulgupta2048** [comment]
Yes, start with that and keep us updated on any challenges you face.
What you should have in mind:-
1. All data, of all communities is to be updated, accurate and clean to the
best of your knowledge. Scour all the places you know to find more
communities.
2. Every community should atleast have a description, their logo, and how
to join with a website link.
3. Improve the UI so that it matches with the rest of the website
(important)
4. Fix #107 along with this.
5. Fix/find more bugs that you see.

**Comment by utkarsh2102** [comment]
Why do we need another page for community?
Why can't we simple re-direct to the community's original site when clicked?

**Comment by vipulgupta2048** [comment]
When clicked what ? Yes, we could. What if they don't have a website.

**Comment by utkarsh2102** [comment]
@vipulgupta2048, most of the communities have a website now. 
If they don't, let's talk to the head and get it done? :/

@thisisayush, every section has a `About Us` section.
They do get a very clear idea after reading the same.
Plus, we do keep posting about the meetups from time-to-time. So doesn't it becomes an unnecessary page to keep?

**Comment by vipulgupta2048** [comment]
I would rather go for having the Communities page as a person being new to ALiAS and open-source who would need quick reference of all communities outside of Amity then finding exceptions of why the page is unnecessary.

Websites take time, care and a lot of patience to get built in Communities and especially open-source. Go ahead if you want to volunteer for the task. I see you already have some experience building a website for Pyladies Delhi. 

Solution - Fix the page

**Comment by thisisayush** [comment]
Validating Tasks to be done and tasks done in #177 

- [ ] Add Mozilla Delhi Open Community [https://wiki.mozilla.org/India/Delhi]
- [ ] Add PyData Delhi Community
- [ ] Add Women Who Go Delhi
- [ ] ILUG-D has changed its logo (Need to be updated)
- [ ] ILUG-D, strangely enough, has the same description as PyDelhi. Change it (Time to git blame)
- [ ] PyDelhi [Description updated here]
- [ ] Improve the Landing page to be in sync with the theme of the entire website [Critical]
- [ ] Mozilla Delhi has links same as ILUGD.

**Comment by Sc0rpi0n101** [comment]
Adding PyData Delhi to #217 and closing this
